docs/content/en/docs/getting-started/install/_index.md

```diff
@@ -79,9 +83,4 @@ You can verify your installed version with
 eksctl anywhere version
 ```
 
-## Deploy a cluster
-
-Once you have the tools installed you can deploy a local cluster or production cluster in the next steps.
-
-* [Create local cluster](../local-environment/)
-* [Create production cluster](../production-environment/)
```

Comment:
We shouldn't remove this section from the original doc without replacing it because it's useful for our standard docs flow.